### DISCUSSION TITLE
Migrate database into production

### DISCUSSION THREAD

**Discussion by pedrolaxe** [strapi/strapi#5914] [Legacy Topics]
Hi, how do I make or migrate the Strapi/Mongo database to put into production only changed data in the commit? any tool for that?

**Comment by TGChrisRArendt** [comment]
Is there a button somewhere to migrate? I feel like it seems pretty obvious that this feature should exist since there is a panel for dbs per environment.. but how do you transfer your data or models from one environment to the other? 

Use case: our client wants to be able to push an update to a page to production from staging. In Wordpress you can just select status such as published. (I know that is more simple because it just switches the value in the db whereas this requires syncing dbs..) I figured we at least want to provide:

1. press button to copy production to staging
2. the client makes their changes
3. changes are qa'd / pass
4. press button to copy staging to production

I really don't want to write a script to traverse every type/row and copy them to our production db. Plus there's the adding a button somewhere in strapi gui which I'm not sure is even possible for me to do..

**Reply by pedrolaxe** [reply]
So I would like to know exactly that, I would like to migrate only the changed data to the production database, how could it be done?

Use case
Posts -> created new posts
import for production only what does not exist in posts

**Reply by TGChrisRArendt** [reply]
Yes I want to know too! :)
